src/layout/GridList.jsx

```diff
@@ -2,6 +2,8 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import cx from 'classnames';
 
+export const GRID_AUTOHEIGHT_CLASS = 'gridList--autoheight';
```

Comment:
stupid nitpick, but could we change the prop name and class name to camel case `autoHeight`? This fits the convention we've been using for multi word prop names.